docs/access-sgts-services-using-techpass.md

```diff
@@ -26,13 +28,13 @@ This article guides you on how to use TechPass to access SGTS resources.
 
 ?> To log out of TechPass, click **Log Out** and confirm which account you would like to log out.
 
-## To log in with TechPass as public officer
+## Log in as public officer
 
 1. Go to the [Developer portal](https://docs.developer.tech.gov.sg/).
 <kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
 2. Sign in to your TechPass account or choose your TechPass account.
 <kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
-3. If you are public officer accessing this SGTS service from your GMD, you will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 on your Authenticator app.
+3. If you are accessing this SGTS service from your GMD, you will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 on your Authenticator app.
```

Comment:
Suggestion:
"... Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app. "

Reply:
Done, thank you!